modules/card.js

```diff
@@ -60,10 +60,19 @@ class MtgCardLoader {
                     response += "\n\n:arrow_right: Other matching cards: :large_blue_diamond:" + otherCardNames.join(" :large_blue_diamond:");
                 }
                 if (card.imageUrl) {
-                    msg.channel.sendFile(card.imageUrl, _.snakeCase(_.deburr(card.name)) + ".jpg", response);
```

Comment:
I would like to keep it this way until, a new version of discord.js is released which handles the problem.

With the current version of discord.js card search would be non-functional while gatherer is down.